spec/shadow/index.html

```diff
@@ -1139,6 +1141,27 @@
           </li>
         </ol>
 
+        <p>Likewise, the <a>sequential focus navigation order</a> for <a>distributed nodes</a>
```

Comment:
Thanks.

Probably we want to start using '[control group](https://html.spec.whatwg.org/multipage/interaction.html#control-group)' (defined in HTML spec) for the scope.
I am working on rewriting with it.